feature/experimental/p4rt/otg_tests/google_discovery_protocol_packetout_lag_test/google_discovery_protocol_packetout_lag_test.go

```diff
@@ -0,0 +1,593 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package google_discovery_protocol_packetout_test
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net"
+	"sort"
+	"testing"
+	"time"
+
+	"google3/base/go/flag"
+
+	"google3/third_party/golang/github_com/cisco_open/go_p4/v/v0/p4rt_client/p4rt_client"
+	"google3/third_party/golang/github_com/cisco_open/go_p4/v/v0/utils/utils"
+	"google3/third_party/golang/gopacket/gopacket"
+	"google3/third_party/golang/gopacket/layers/layers"
+	"google3/third_party/golang/ygot/ygot/ygot"
+	"google3/third_party/open_traffic_generator/gosnappi/gosnappi"
+	"google3/third_party/openconfig/featureprofiles/internal/deviations/deviations"
+	"google3/third_party/openconfig/featureprofiles/internal/fptest/fptest"
+	"google3/third_party/openconfig/featureprofiles/internal/otgutils/otgutils"
+	"google3/third_party/openconfig/featureprofiles/internal/p4rtutils/p4rtutils"
+	"google3/third_party/openconfig/ondatra/gnmi/gnmi"
+	"google3/third_party/openconfig/ondatra/gnmi/oc/oc"
+	"google3/third_party/openconfig/ondatra/netutil/netutil"
+	"google3/third_party/openconfig/ondatra/ondatra"
+	p4v1pb "google3/third_party/p4lang_p4runtime/proto/p4runtime_go_proto"
+)
+
```

Comment:
These library path should be from be from `github.com/` instead of `google3/third_party/`